### PR TITLE
log exception as "error" since we are retrying anyways

### PR DESCRIPTION
**Pull request by stishkin**

### CODE REVIEW

File: src/ApiService/ApiService/onefuzzlib/WebhookOperations.cs [hunk 1]
```diff
@@ -269,7 +269,7 @@ public WebhookMessageLogOperations(ILogger<WebhookMessageLogOperations> log, IOn
             }
             return sendResult.IsOk;
         } catch (Exception exc) {
-            _logTracer.LogError(exc, "Send Webhook");
+            _logTracer.LogError("Send Webhook: {exception}", exc);
```
Comment (mgreisen) [inline src/ApiService/ApiService/onefuzzlib/WebhookOperations.cs:272]: Does the retry code eventually throw an exception if we exhaust retries?

Comment (stishkin) [reply src/ApiService/ApiService/onefuzzlib/WebhookOperations.cs:272]: No, we just log a warning:

_logTracer.LogWarning("sending webhook event failed, re-queued {WebhookId}:{EventId}", newMessage.WebhookId, newMessage.EventId);

Comment (stishkin) [reply src/ApiService/ApiService/onefuzzlib/WebhookOperations.cs:272]: Good point though, we can log an exception so alerting picks this up

**Review by mgreisen — APPROVED** [review]
:shipit:

**Pull request merged**